tfhe/examples/sha256.rs

```diff
@@ -37,7 +35,9 @@ macro_rules! __join_implementation {
     };
 }
 
-#[macro_export]
+#[doc(hidden)]
```

Comment:
doc(hidden) on the re export ?

Reply:
It was already hidden, but I can remove it if you think it makes sense

Reply:
I mean does a doc hidden make sense on a pub use ?

Reply:
It's a `pub(crate) use`

Reply:
sorry

I mean

doc(hidden) on a use statement does nothing ?

Reply:
Removed